ui/desktop/src/components/pair.tsx

```diff
@@ -80,6 +80,20 @@ export default function Pair({
   // Handle initial message from hub page
   useEffect(() => {
     const messageFromHub = location.state?.initialMessage;
+    const resetChat = location.state?.resetChat;
+
+    // If we have a resetChat flag from Hub, clear any existing recipe config
```

Comment:
I don't follow the logic here. in what scenario is this going to be triggered? there was a recipe, the user went to the hub ...